### PR TITLE
Trans fr

### LINKED ISSUE #427: New appimage (2.0.1) is no more localised
Everything is in English, all the translation are missing (French for me).

### PR DESCRIPTION
**Pull request by mgautierfr**
Fix #427

### CODE REVIEW

File: resources/i18n/fr.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
+{
+    "@metadata": {
+		"authors": [
+		]
+	},
+    "name":"Français",
+    "error-downloader-window-title":"Impossible de lancer le system de téléchargement",
+    "error-downloader-launch-message":"Impossible de lancer le system de téléchargement, Kiwix-desktop fonctionnera mais sans les fonctionnalités de téléchargement !",
```
Comment (jetownfeve21) [inline resources/i18n/fr.json:8]: small error `system` instead of `système`

File: resources/i18n/fr.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    "close-tab":"Fermer l'onglet",
+    "reopen-closed-tab":"Rouvrir l'onglet fermé",
+    "browse-library":"Parcourir la bibliothèque",
+    "open-file":"Ouvrir un fichier",
+    "open-recent":"Fichiers récemment ouverts",
+    "search-article":"Chercher un article",
+    "search-in-library":"Chercher dans la bibliothèque",
+    "find-in-page":"Chercher dans la page",
+    "set-fullscreen":"Mode plein écran",
+    "quit-fullscreen":"Quitter le plein écran",
```
Comment (jetownfeve21) [inline resources/i18n/fr.json:24]: Maybe `Quitter le mode plein écran` ?

Comment (mgautierfr) [reply resources/i18n/fr.json:24]: I've use the translation in previous translation file :

https://github.com/kiwix/kiwix-desktop/commit/ffdfff514e169eea42ba7b3d86b48f01120c505e#diff-85adf1c107d0d00b2b7d1570fafc5a70L106

File: resources/i18n/fr.json (hunk 1 above)
Comment (mgautierfr) [reply resources/i18n/fr.json]: Fixed

**Pull request merged**